src/components/Navbar.js

```diff
@@ -10,6 +10,7 @@ const Navbar = () => (
     </div>
     <div className="Navigation-Links">
       <ul className="Link-List">
+        <li><NavLink to="/" activeClassName="Link-Active" className="Link-NoActive">Rockets</NavLink></li>
```

Comment:
When I click in the other links the Rockets links stays active. Kindly fix that.
One way you can try to achieve this is by adding exact to NavLink. For example:


`<NavLink exact to="/"></NavLInk>`.


![review](https://user-images.githubusercontent.com/72808808/132306638-e11bf33d-8192-41a3-9241-209995f1e6c1.png)
 
Thank you

Reply:
Thank you for your feedback.
The problem is resolved.